scripts/sct_fmri_moco.py

```diff
@@ -6,29 +6,26 @@
 # ---------------------------------------------------------------------------------------
 # Copyright (c) 2013 Polytechnique Montreal <www.neuro.polymtl.ca>
 # Authors: Karun Raju, Tanguy Duval, Julien Cohen-Adad
-# Modified: 2014-10-06
 #
 # About the license: see the file LICENSE.TXT
 #########################################################################################
 
-# TODO: estimate and apply, no need to reapply afterwards
 
 from __future__ import absolute_import, division
 
 import sys
 import os
-import getopt
 import time
 import math
-
+import shutil
```

Comment:
There is a `copy()` function in `sctutils`

Reply:
Fixed in 44e02e4e44c74da51cfb2cc9a95eb025155cd87a